src/odemis/acq/fastem.py

```diff
@@ -466,6 +470,16 @@ def pre_calibrate(self):
             logging.debug("Read initial Hw settings.")
             asm_config = configure_hw.get_config_asm(self._multibeam, self._descanner, self._detector)
 
+            # temp fix: hardcode scan amplitude
+            asm_config["multibeam"]["scanAmplitude"] = configure_hw.CALIBRATED
+            asm_config["multibeam"]["scanOffset"] = configure_hw.CALIBRATED
+            configure_hw.configure_asm(self._multibeam,
+                                       self._descanner,
+                                       self._detector,
+                                       self._detector.data,
+                                       asm_config,
+                                       upload=False)
+
```

Comment:
Do we want to have this on master, or should it just be a temp commit on EA2?

Reply:
yes let's make it a separate temporary commit.